doc/build_apps/js_app_bundle.rst

```diff
@@ -13,12 +13,7 @@ This page documents the components of a bundle and the JavaScript API available
     `TypeScript <https://www.typescriptlang.org/>`_.
     CCF provides an example app built with these tools.
     They involve a `build` step that generates an app bundle suitable for CCF.
-    See the following for more details:
-
-    .. toctree::
-      :maxdepth: 1
-
-      js_app_ts
+    See the :ref:`TypeScript Application <build_apps/js_app_ts>` section for more details
```

Comment:
This additional `toctree` caused `TypeScript Application` to be referenced twice:

![image](https://user-images.githubusercontent.com/6000239/149921780-a4559bd7-63c0-4095-8765-d923c3720c41.png)

Reply:
I'm not convinced of the benefits of one more level of depth here, I think two levels of toctree is enough.

Reply:
Sure, I'll raise both of these to go under `Build Apps`.